ProductService/src/functions/getProductsById/index.ts

```diff
@@ -0,0 +1,14 @@
+import { handlerPath } from '@libs/handlerResolver';
+
+export default {
+  handler: `${handlerPath(__dirname)}/handler.main`,
+  events: [
+    {
+      http: {
+        method: 'get',
+        cors: true,
+        path: 'products/getProductsById/{id}',
```

Comment:
This is not followed rest principle to name resources. It's better to have names like: `/produtcs` to get products and `/products/{id}` to get product by id. In rest you operate resources not actions like `getProductsById`. In order you need to get resource you should send `GET /resource` or to delete resource `DELETE /resource`.

Reply:
OK, thanks. I misunderstood the task, I thought that the names of the lambdas must match the path